citi-onboarding/PubliEasy
# Feature/nodemailer back correct

## What I did

- Coloquei o email de teste como o meu 
- Defini as colunas como name, number, message e email
   - Conectei com o insomia
   - Coloquei o subjectText, destination, html do email
   - Mudança no env. já que tive que rodar localmente


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/nodemailerBackCorrect`
   1. `git pull origin feature/nodemailerBackCorrect`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 4h30min

**Resolves:** #<issue number>
